spec/lib/field-types.js

```diff
@@ -6,16 +6,16 @@ const overwrite = (schema, opts = {}) => Overwrite({ valueSchema: schema, ...opt
 module.exports = {
   string: overwrite({ type: ['string', 'null'] }),
   poBox: overwrite({
-    type: 'object',
+    type: ['object', 'null'],
     required: ['publicKey', 'secretKey'],
     properties: {
       publicKey: {
-        type: ['string', 'null'],
+        type: ['string'],
         pattern: '^ssb://'
       },
-      privateKey: {
-        type: ['string', 'null'],
-        pattern: isBase64
+      secretKey: {
```

Comment:
Took me ages to realise that i had a stupid typo here :/ was wondering why it kept passing even though it was wrong ugh